### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by rodinux**
# Problem

-  *Useless redirection in nginx.conf*

# Solution

-  *Delete useless redirections in nginx.conf*

### CODE REVIEW

**Review by alexAubin — APPROVED** [review]
Okay I guess ?

**Review by alexAubin — APPROVED** [review]

**Comment by rodinux** [comment]
Hello, Since this PR, the upgrade of the version is now to 1.0.3. It would be nice to merge with the master branch...

**Comment by alexAubin** [comment]
Eeeeeh yes it's not clear who's the maintainer of the app ... I thought you were ... If you're not confident or don't feel legitimate to click the merge button then feel free to complain and we can do this together ;)

File: pull_request_template.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
-- [x] Tested with Package_check.
-- [x] Fix or enhancement tested.
-- [x] Upgrade from last version tested.
-- [x] Can be reviewed and tested.
+- [ ] Tested with Package_check.
+- [ ] Fix or enhancement tested.
+- [ ] Upgrade from last version tested.
+- [ ] Can be reviewed and tested.
 
 ## Package_check results
```
Comment (alexAubin) [inline pull_request_template.md:15]: Well uuuuh you're not supposed to put actual stuff in this file ... this is a template automatically used by Git when somebody creates a PR ..

**Review by alexAubin — APPROVED** [review]
Super quick review, LGTM apart from the change on pull_request_template.md which should just be a copypasta of the one in example_ynh..

File: pull_request_template.md (hunk 1 above)
Comment (BenoitCier) [reply pull_request_template.md]: !testme

**Comment by rodinux** [comment]
!testme

**Pull request merged**